### ISSUE TITLE
Add ability to change light mode

### ISSUE THREAD

**Issue by beeradmoore** [OnewheelCommunityEdition/OWCE_App#36]
App currently does not allow you to turn your lights on or off.

**Comment by onewheeler70** [comment]
I would like this feature in the app, and have an Android "Intent" that I can call from my Flic bluetooth button. This would allow me to turn off the light when approaching walkers, as the light is too bright and I can see they have trouble seeing me. The Intent allows me to use a bluetooth button that I can clip to my shirt or helmet.

**Comment by beeradmoore** [comment]
I did not realise there were devices we can communicate to an app with via intents. This would be a nice little addition for sure.

**Comment by onewheeler70** [comment]
I use a Flic button "https://flic.io". It is a bluetooth button, and the software allows me to call an Intent in any software upon clicking the button. A nice way to perform a software action by clicking a hardware button away from the phone. I am not sure if this OWCE app is still in active development. I would offer to help out, but Xamarin is not one of my software skillsets. Will have a look at learning it.

**Comment by beeradmoore** [comment]
Development is semi-active, but I am super-active when it comes to feedback, pointing people in the right direction, etc.

In terms of the bluetooth+intent stuff it would be within the Android side so large parts of it would still be normal Android code, just in C# not Java.

Light toggle work is not done yet but it could be a good starter to understand the project structure and interconnects. Seeing the correct characteristic to change and how to change it can be discovered organically or you can see how [POnewheel](https://github.com/ponewheel/android-ponewheel) does it.

As for everything else, I don't know if we need to have any UI for the user to enable intents. If there were other intents for things like ride mode I imagine we would not want that to be togglable by every app? (Although ride mode does nothing while the device is in motion)

**Comment by onewheeler70** [comment]
Yeah, good question on ride mode toggling via intent. I wondered if there was anyone who might want to toggle it at the start of a large hill and flick back after. No idea on the safety of that, and I guess you are saying that the board will deny the change anyway. If the board denies it safety-first, then I guess the intent could be there. Am a little busy ATM, but I will check out the code and get a feel for the work.
